appendix/templates/login.html

```diff
@@ -0,0 +1,10 @@
+{% extends "templates/login.html" %}
+{% block site %}
+
+<div id="ipython-main-app" class="container">
+    <h1>You don't have access to this Binder!</h1>
```

Comment:
! -> . ? (change ! to . maybe?)

Other ideas that might work: "This isn't the binder you are looking for." or "You can't access this Binder." or "This Binder belongs to someone else."